go/client/cmd_chat_api.go

```diff
@@ -150,7 +169,7 @@ func (c *CmdChatAPI) ListV1(ctx context.Context) Reply {
 			},
 		}
 	}
-
+	cl.RateLimits.RateLimits = c.aggRateLimits(rlimits)
```

Comment:
can this just be 

    cl.RateLimits.RateLimits = c.aggRateLimits(inbox.RateLimits)

?